src/extensions/cookie/bootstrap-table-cookie.js

```diff
@@ -35,135 +35,42 @@ const UtilsCookie = {
 
     return searchControls
   },
-  cookieEnabled () {
+  isCookieSupportedByBrowser () {
     return !!(navigator.cookieEnabled)
   },
-  inArrayCookiesEnabled (cookieName, cookiesEnabled) {
-    let index = -1
-
-    for (let i = 0; i < cookiesEnabled.length; i++) {
-      if (cookieName.toLowerCase() === cookiesEnabled[i].toLowerCase()) {
-        index = i
-        break
-      }
-    }
-
-    return index
+  isCookieEnabled (that, cookieName) {
+    return that.options.cookiesEnabled.indexOf(cookieName) !== -1
```

Comment:
that.options.cookiesEnabled.includes(cookieName)

Reply:
Thanks